src/App.tsx

```diff
@@ -20,12 +22,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
```

Comment:
it's a bad practice to use arrow functions like that. move it to a separate function